app/gui/src/integration_test.rs

```diff
@@ -104,6 +120,88 @@ impl Drop for Fixture {
 }
 
 
+// === Profiling objectives ===
+
+impl Fixture {
+    /// Create a new project; doesn't complete until the project is ready to render.
+    #[profile(Objective)]
+    pub async fn create_project() -> Self {
+        let test = Self::setup_new_project().await;
+        test.compile_new_shaders().await;
+        test
+    }
+
+    /// Open a project; doesn't complete until the project is ready to render.
+    #[profile(Objective)]
+    pub async fn open_project() -> Self {
+        let template_project = "Orders";
```

Comment:
This could probably have a better name and comment indicating that "Orders" is the default project used.